### ISSUE TITLE
Feature request: refer to a Group rather than a list of repos

### ISSUE THREAD

**Issue by nigelmegitt** [dontcallmedom/github-notify-ml#23]
It would be helpful to track all the repos associated with a Group so that when new repos are created they automatically get added into the notifier without an extra maintenance task.

**Comment by richsalz** [comment]
Yes, something like "allRepos: name" where `name` names a GitHub organization, and it queries to get all the repo's in that organization from `https://github.com/orgs/NAME/repositories`

Would you be interseted in a PR that adds this?

**Comment by nigelmegitt** [comment]
I think in the case of W3C Groups, they are all in the same GitHub organization. There's a finer level of granularity that associates each repo with a Group.

**Comment by dontcallmedom** [comment]
@richsalz I think a PR that would work well (and which I could likely easily build on to cater for W3C groups specificities) would be one that fetch a JSON file to figure the list of repos to watch, maybe with room for a simple declarative massaging of the JSON into a suitable list of repo names.

**Comment by richsalz** [comment]
Thanks, I'll poke at it.  Probably need allGitHubRepo's or some such to avoid confusing with W3C.
